src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -6741,16 +6718,6 @@ private NameSyntax ParseQualifiedNameRight(
             }
         }
 
-        private NameSyntax RecoverFromDotDot(NameSyntax left, ref SyntaxToken separator)
```

Comment:
instead of producing a `..` and breaking it up, we produce `.` `.`, and only particular parts of the parser merge those.